### PR TITLE
Ryan

### PR DESCRIPTION
**Pull request by RyanSusana**
Fixed image, added css

### CODE REVIEW

File: layouts/partials/team.html [hunk 1]
```diff
@@ -63,7 +63,7 @@ <h5 class="modal-title" id="{{ $id }}Title">{{ .name }}</h5>
                     </div>
                     <div class="modal-body">
                       {{ .description | markdownify }}
-                      <img src="{{ $image300.RelPermalink }}" class="img-fluid">
+                      <img src="{{ $image300.RelPermalink }}" class="img-fluid mt-3">
```
Comment (bennycornelissen) [inline layouts/partials/team.html:66]: Does the `mt-3` part do anything special? 

(If it makes me look better then LGTM 😛 )

Comment (RyanSusana) [reply layouts/partials/team.html:66]: Adds some spacing between the text and the image (margin-top) this is too little:
image

Comment (bennycornelissen) [reply layouts/partials/team.html:66]: Ah I see. And I agree the extra spacing looks better. Can you also add it on line 133 (Alumni section) of the same file then? 😄

Comment (bennycornelissen) [reply layouts/partials/team.html:66]: I can also add it in in the rework I'm doing of the alumni section. I'll just merge this. 👍

**Review by bennycornelissen — APPROVED** [review]
LGTM!

**Pull request merged**